src/pywifes.py

```diff
@@ -1665,7 +1665,7 @@ def wifes_slitlet_mef(inimg, outimg, data_hdu=0,
         nslits = 12
         #offset = 4096//bin_y
         old_ymin = int(float(
-            old_hdr['CCDSEC'].split(',')[1].split(':')[0]))-1
+            old_hdr['DETSEC'].split(',')[1].split(':')[0]))-1
```

Comment:
Definitely not something to change now, but I'm seeing a few of these `.split(',')[1].split(':')[0]))`  statements. 

It may be worth making a function called `get_ystart(detsec)`, and calling that each time. I think it's best to wait for https://github.com/PyWiFeS/pipeline/pull/24/files at least.